src/__tests__/ContextMenu.spec.tsx

```diff
@@ -39,10 +40,11 @@ describe('ContextMenu component', () => {
     const trigger = 'I am a trigger!';
     const renderTrigger = () => trigger;
 
-    const wrapper = shallow(<ContextMenu renderTrigger={renderTrigger} id="uniq-id" />);
+    const wrapper = mount(<ContextMenu renderTrigger={renderTrigger} id="uniq-id" />);
```

Comment:
Maybe just fake `useRef` implementation? We can get rid of it once shallow rendering is supported/